### PR TITLE
LATTE: Add version 1.2.2.

### PR DESCRIPTION
**Pull request by RemiLacroix-IDRIS**

### CODE REVIEW

**Review by junghans — APPROVED** [review]

File: var/spack/repos/builtin/packages/latte/package.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class Latte(CMakePackage):
     tags = ['ecp', 'ecp-apps']
 
     version('develop', branch='master')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/latte/package.py]:
```suggestion
    version('master', branch='master')
```

**Review by alalazo — APPROVED** [review]

File: var/spack/repos/builtin/packages/latte/package.py (hunk 1 above)
Comment (RemiLacroix-IDRIS) [reply var/spack/repos/builtin/packages/latte/package.py]: I am having second thoughts about this. Out of curiosity I checked and I found around 150 occurrences of "develop" version being tied to the "master" branch and only around 60 of "develop" actually being tied to a "develop" branch.

I forgot to check how many "master" branch are used as the "master" version though.

Is there some kind of general policy regarding this?

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/latte/package.py]: The reason for this is that once upon a time, we treated develop as newer than all releases, and master as older than all releases. This bug was fixed years ago, but there are still tons of packages that haven't been updated yet. The policy is to have them match to the best of our ability.

Comment (RemiLacroix-IDRIS) [reply var/spack/repos/builtin/packages/latte/package.py]: Ok, got it.

**Pull request merged**